packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -19,6 +19,7 @@ interface MenuProps {
   onOptionSelect(chosenOption: Option): void;
 }
 
+// eslint-disable-next-line max-statements
```

Comment:
How many statements are we at? We should avoid doing this if we can. I think we have a few opportunities to simplify the logic in here.

Reply:
Yup, totally! Simplified logic and removed max statement es-lint disable